### ISSUE TITLE
Possible Performance Improvements

### ISSUE THREAD

**Issue by Tschipp** [wolforcept/Vaultopic#4]
![image](https://user-images.githubusercontent.com/11649893/64493976-56c9e280-d287-11e9-9a66-7fff26c3b596.png)
Searching things in chests that appear ferquently causes big frame drops. I'm not sure if you can do anything to improve this, but if you could that would be nice.

**Comment by wolforcept** [comment]
Jesus nice screenshot.
Anyway, i think that for those huge ammounts of items this mod isnt really what you want to be using. Its more for early inventory management, there are other mods that could be better for that scenario. 
The reason there are framedrops is probably because i had to make all the math to know where the items are on tje screen so that you could click on them,they are not how usually items are rendered. So that might be it.

**Comment by Tschipp** [comment]
Yeah, that's probably it. If I have some spare time I'll take a look at your code and see how you did it.
